components/dashboard/src/components/forms/CheckboxInputField.tsx

```diff
@@ -28,8 +28,8 @@ type CheckboxInputProps = {
     value: string;
     checked: boolean;
     disabled?: boolean;
-    label: string;
-    hint?: string;
+    label: string | React.ReactNode;
+    hint?: string | React.ReactNode;
```

Comment:
**thought:** Not sure about this change, but let me loop in @selfcontained who initially added this component.

Reply:
This change is made to accommodate for the usage of `<CheckBox />` component in certain places (like [`ProjectSettings`](https://github.com/gitpod-io/gitpod/pull/16813/files#diff-5d008ce9669cd9e22186bc906fca1d6263f6cdb81cd3e4e2a099a71ab48c6083L129-L138)) where React nodes are passed as props to specific attributes instead of text.

Reply:
Makes sense, thanks! We can simplify it to just the `React.ReactNode` type as that allows for the `string` type as well.